hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/Constants.java

```diff
@@ -41,12 +41,17 @@
    * then the hadoop default value (/user) is used.
    */
   public static final String CONFIG_VIEWFS_HOMEDIR = "homedir";
-  
+
+  /**
+   * Config key to specify the name of the default mount table.
+   */
+  public static final String CONFIG_VIEWFS_DEFAULT_MOUNT_TABLE_NAME_KEY = "fs.viewfs.mounttable.default.name.key";
```

Comment:
You may want to add some documentation in ViewFSOverloadScheme.md docs?

Reply:
Added details in ViewFSOverloadScheme.md.